src/routes/docs/products/messaging/messages/+page.markdoc

```diff
@@ -35,8 +35,8 @@ Each message displays with the following information.
 * Type
 * Type of message, either `Push`, `Email`, and `SMS`.
 ---
-* Status
-* Indicates the status of the message, can be one of `draft`, `scheduled`, `processing`, `failed`, `success`.
+* Draft
+* If the message is a draft or ready to be sent. Can be `true` or `false`.
```

Comment:
Draft isn't shown in the table. The status is still shown

Reply:
Something's wrong with render againnnnn

Reply:
Sorry, I meant in the Console, there is no Draft column:

<img width="1185" alt="image" src="https://github.com/appwrite/website/assets/1477010/b32dbc15-31c5-47e3-bd13-57836521cb56">

So the previous text was fine.